test/test_module_init.py

```diff
@@ -366,6 +366,7 @@ def generate_tests(test_cls, constructor_arg_db):
         # See https://github.com/pytorch/pytorch/issues/55396
         torch.nn.quantized.Embedding,
         torch.nn.quantized.EmbeddingBag,
+        torch.nn.quantized.modules.rnn.LSTM,
```

Comment:
nit: torch.nn.quantized.LSTM (if it doesn't work we should import the module in https://github.com/pytorch/pytorch/blob/master/torch/nn/quantized/modules/__init__.py

Reply:
this should be fixed before landing